tests/zfs-tests/tests/functional/inheritance/inherit_001_pos.ksh

```diff
@@ -331,7 +331,7 @@ function scan_state { #state-file
 				if [[ $op == "-" ]]; then
 					log_note "No operation specified"
 				else
-					export __ZFS_POOL_RESTRICT="$TESTPOOL"
+					export __ZFS_POOL_RESTRICT="TESTPOOL"
```

Comment:
@jwk404 Does inherit_001_pos work for OpenZFS? I noticed that this is the only place "TESTPOOL" was being expanded.  It appears that else where in the script, the pool name is expected to be "TESTPOOL".